app/modules/sightings/models.py

```diff
@@ -1073,6 +1075,12 @@ def _ensure_annot_data_in_response(self, annot, response):
                 'image_url': annot.asset.get_image_url(),
                 'asset_dimensions': annot.asset.get_dimensions(),
                 'bounds': annot.bounds,
+                'sighting_guid': self.guid,
+                'sighting_time': self.get_edm_data_field('time'),
```

Comment:
these are _not_ edm fields -- they are ComplexDateTime values within houston, e.g. `enc.time` and `sighting.time`.  there are some helper functions to "unwrap" the complexity of ComplexDateTime that will be useful here.  specifically, see `enc.get_time_isoformat_in_timezone()` and `enc.get_time_specificity()` (and same on Sighting), which will also handly the null-case.  (tho `.time` is required on Sighting; but can be null on Encounter)